packages/textfield/README.md

```diff
@@ -213,6 +213,15 @@ Inherits CSS Custom properties from:
 | `--mdc-text-field-outlined-idle-border-color`     | `rgba(0, 0, 0, 0.38)` | Color of the outlined textfield's  outline when idle.
 | `--mdc-text-field-outlined-hover-border-color`    | `rgba(0, 0, 0, 0.87)` | Color of the outlined textfield's outline when hovering.
 | `--mdc-text-field-outlined-disabled-border-color` | `rgba(0, 0, 0, 0.06)` | Color of the outlined textfield's outline when disabled.
+| `--mdc-text-field-fill-color`                     | filled: `rgb(245, 245, 245)`, outlined: `transparent` | Color of the textfield's background fill.
```

Comment:
Is it within the spec to have an outlined textfield without a transparent background?

Reply:
nothing in the spec, but reading over the mdc docs, this is not a thing they expose. Removing it.